src/NuGet.Clients/NuGet.CommandLine/Commands/HelpCommand.cs

```diff
@@ -91,13 +91,16 @@ orderby c.CommandAttribute.CommandName
         // Help command always outputs NuGet version
         protected override bool ShouldOutputNuGetVersion { get { return true; } }
 
-        private void PrintCommand(int maxWidth, CommandAttribute commandAttribute)
+        private void PrintCommand(int maxWidth, CommandAttribute commandAttribute, DeprecatedCommandAttribute deprecatedCommandAttribute = null)
```

Comment:
`PrintCommand` is only used in a single place, where it passes a value for `deprecatedCommandAttribute`, so there's no benefit to making it an optional parameter.